.github/workflows/ci_cd_release.yml

```diff
@@ -23,197 +22,40 @@ jobs:
-            LICENSE_SERVER: ${{ secrets.LICENSE_SERVER }}
-        run: |
-          docker run \
-            --detach -it \
-            --network="host" \
-            --name ${DOCKER_CONTAINER_LUMERICAL_UNIFIED_NAME} \
-            --env ANSYSLMD_LICENSE_FILE=1055@${LICENSE_SERVER} \
-            --mount type=bind,source=${PWD},target=/home/lumerical/pylumerical \
-            ${DOCKER_IMAGE_LUMERICAL_UNIFIED}
-
-      - name: "Install Project Dependencies and Run Tests"
-        env:
-          DOCKER_CONTAINER_LUMERICAL_UNIFIED_NAME: ${{ env.DOCKER_CONTAINER_LUMERICAL_UNIFIED_NAME }}
-        run: |
-            docker exec \
-              --workdir /home/lumerical/pylumerical \
-              ${DOCKER_CONTAINER_LUMERICAL_UNIFIED_NAME} /bin/bash --login -c \
-              "python -m pip install -e .[tests] && \
-              pytest --cov="/home/lumerical/.local/lib/python3.13/site-packages/ansys/api/lumerical/" \
-              --cov="./tests/unit" --cov-report=xml:.cov/xml --cov-report=html:.cov/html --verbose"
-
-      - name: "Restore original ownership"
-        if: always()
-        env:
-          ORIG_UID: ${{ env.ORIG_UID }}
-          ORIG_GID: ${{ env.ORIG_GID }}
-        run: |
-          if [[ -z "$ORIG_UID" || -z "$ORIG_GID" || ! "$ORIG_UID" =~ ^[0-9]+$ || ! "$ORIG_GID" =~ ^[0-9]+$ ]]; then
-            if [[ -z "$ORIG_UID" || ! "$ORIG_UID" =~ ^[0-9]+$ ]]; then
-              echo "Skipping chown: ORIG_UID ('$ORIG_UID') is not set or not a valid integer (expected a numeric UID)."
-            elif [[ -z "$ORIG_GID" || ! "$ORIG_GID" =~ ^[0-9]+$ ]]; then
-              echo "Skipping chown: ORIG_GID ('$ORIG_GID') is not set or not a valid integer (expected a numeric GID)."
-            else
-              echo "Skipping chown: Unknown error with ORIG_UID ('$ORIG_UID') or ORIG_GID ('$ORIG_GID')."
-            fi
-          else
-            sudo chown -R "$ORIG_UID:$ORIG_GID" .
-          fi
-          ls -la
-
-      - name: Upload Coverage Results
-        if: always()
-        uses: actions/upload-artifact@ea165f8d65b6e75b540449e92b4886f43607fa02 # v4.6.2
-        with:
-          name: coverage-html-ubuntu
-          path: .cov/html
-          retention-days: 7
-
-      - name: Upload coverage to Codecov
-        uses: codecov/codecov-action@5a1091511ad55cbe89839c7260b706298ca349f7 # v5.5.1
-        env:
-          CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
-        with:
-          files: .cov/xml
-
-      - name: "Stop lumerical services"
-        if: always()
-        env:
-          DOCKER_CONTAINER_LUMERICAL_UNIFIED_NAME: ${{ env.DOCKER_CONTAINER_LUMERICAL_UNIFIED_NAME }}
-        run: |
-          docker stop ${DOCKER_CONTAINER_LUMERICAL_UNIFIED_NAME}
-          docker logs ${DOCKER_CONTAINER_LUMERICAL_UNIFIED_NAME}
-          docker rm ${DOCKER_CONTAINER_LUMERICAL_UNIFIED_NAME}
-
-  package:
-    name: Package library
-    needs: [tests, doc-build]
```

Comment:
Same for the package stage - this one is necessary for the releasing to go through